as/itests/org.jboss.tools.as.ui.bot.itests/src/org/jboss/tools/as/ui/bot/itests/SmokeTestsSuite.java

```diff
@@ -0,0 +1,15 @@
+package org.jboss.tools.as.ui.bot.itests;
+
+import org.eclipse.reddeer.junit.runner.RedDeerSuite;
+import org.jboss.tools.as.ui.bot.itests.parametized.server.ShowInBrowserTest;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+@RunWith(RedDeerSuite.class)
+@Suite.SuiteClasses({
+	ShowInBrowserTest.class
```

Comment:
ShowInBrowser.class should be in https://github.com/jbosstools/jbosstools-server/blob/master/as/itests/org.jboss.tools.as.ui.bot.itests/src/org/jboss/tools/as/ui/bot/itests/AllTestsSuite.java